Source/WebCore/inspector/agents/InspectorCSSAgent.cpp

```diff
@@ -414,56 +414,56 @@ bool InspectorCSSAgent::forcePseudoState(const Element& element, CSSSelector::Ps
     return m_nodeIdToForcedPseudoState.get(nodeId).contains(pseudoClass);
 }
 
-std::optional<Inspector::Protocol::CSS::PseudoId> InspectorCSSAgent::protocolValueForPseudoId(PseudoId pseudoId)
+std::optional<Inspector::Protocol::CSS::PseudoId> InspectorCSSAgent::protocolValueForPseudoId(PseudoElementType pseudoElementType)
```

Comment:
```suggestion
std::optional<Inspector::Protocol::CSS::PseudoId> InspectorCSSAgent::protocolValueForPseudoElementType(PseudoElementType pseudoElementType)
```

We should be able to safely rename this without affecting the protocol